airflow/models/dagrun.py

```diff
@@ -85,6 +87,17 @@ class TISchedulingDecision(NamedTuple):
     finished_tis: list[TI]
 
 
+def _creator_note(val):
+    """Custom creator for the ``note`` association proxy."""
+    # breakpoint()
```

Comment:
Same for the breakpoint comment. Should the name be reversed _note_creator ?

Reply:
<general>_<specific>